app/models/article.rb

```diff
@@ -241,6 +241,12 @@ def self.active_eli5(time_ago)
     stories.pluck(:path, :title, :comments_count, :created_at)
   end
 
+  def self.feed
+    published.
+      select(:id, :published_at, :processed_html, :user_id, :organization_id, :title, :path).
+      order(published_at: :desc)
+  end
+
```

Comment:
code this become an actual scope? see for example https://github.com/thepracticaldev/dev.to/blob/0d2506f9ef79a32f7474001337161314d1062134/app/models/article.rb#L79

I'm a little conflicted about adding ordering to scopes though, what do you think @lightalloy ?

Reply:
It could definitely be a scope. I don't have a preference on ordering. I like to try keeping query specific info out of controllers. Let me know which you prefer and I'll make adjustments.